pkg/port/ports.go

```diff
@@ -0,0 +1,25 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package port
+
+const (
+	NATTDiscovery = 4490
+	UDP           = 4500
```

Comment:
```suggestion
	ExternalTunnel           = 4500
```

As this constant can be used by external clients, lets use a bit more descriptive name as UDP is a protocol in itself.